mxnet-bot/LabelBotPredict/elastic_beanstalk_server/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM python:3.6.6
+
+# Update packages
+RUN apt-get update
+
+# Install prerequisite for matplotlib
+RUN apt-get -y install libxft-dev libfreetype6 libfreetype6-dev
+
+# Bundle app source
+COPY . /src
+
+EXPOSE 8000
+WORKDIR /src
+
+#install Python modules
+RUN pip install -r requirements.txt
+
+# Run it
+ENTRYPOINT ["python", "application.py"]
+
+# Environment Variables
+ENV github_user your_github_id
```

Comment:
Could you please set these before entry point?

Reply:
Done